ndicapimodule.cxx

```diff
@@ -25,6 +25,7 @@
   #define PyString_Format PyUnicode_Format
   #define PyString_AsString PyUnicode_AsUTF8
   #define PyIntObject PyLongObject
+  //#define PY_INT_OBJECT_OB_IVAL(ob) PyLong_AsLong((PyObject*)(ob))
```

Comment:
Please remove commented code. If you want to keep it, please add a comment above as to why it's being kept.

Reply:
The safe way to get the long inside a PyLong is to use PyLong_AsLong(PyObject*) and not just ob_digit[0]. All around in CPython PyLong_AsLong is used. 
https://github.com/python/cpython/blob/6405feecda6a5d5dd7a4240eb3054a2676ed29b1/Objects/longobject.c

Reply:
I think you know more about this than I do. Whatever you think is right (and works on all platforms).